### ISSUE TITLE
Self-closing tags with alias

### ISSUE THREAD

**Issue by idpaterson** [michaelkourlas/node-js2xmlparser#34]
I noticed that it is not possible to create a self-closing tag using an alias, there is always a start and end tag. This is easily resolved by changing the `lengthExcludingAttributes` logic to subtract 1 in these cases:
```js
if (typeof object[property][attributeString] !== "undefined") {
    lengthExcludingAttributes -= 1;
}

if (typeof object[property][aliasString] !== "undefined") {
    lengthExcludingAttributes -= 1;
}
```

However, there are unit tests relying on the current behavior for aliased elements, where `{ '=': 'b' }` is represented as `<b></b>` rather than `<b/>`. I did not submit a pull request since this should certainly be discussed before making a change like that; perhaps an explicit key to designate that the tag is self-closing is needed.

I may be approaching the problem incorrectly. I need to reproduce the following structure:
```xml
<dict>
    <key>foo</key>
    <true/>
    <key>bar</key>
    <false/>
</dict>
```

What I have so far, however, results in `<true></true>` and `<false></false>`:
```js
{
	dict: {
		key: [
			'foo',
			{
				'=': 'true'
			},
			'bar',
			{
				'=': 'false'
			}
		]
	}
}
```

It should work for my use case, but I wanted to point out this issue since special handling was added for tags that have only attributes.

**Comment by michaelkourlas** [comment]
This looks like a bug. I'll take a look.

**Comment by michaelkourlas** [comment]
This is now fixed in 2.0.0.
